packages/falso/src/lib/superhero-name.ts

```diff
@@ -0,0 +1,38 @@
+import { fake } from './core/core';
+import { data } from './superhero.json';
+import { rand } from './rand';
+import { Superhero, SuperheroOptions } from './superhero';
+
+/**
+ * Generate a random superhero name.
+ *
+ * @category comic book
+ *
+ * @example
+ *
+ * randSuperheroName()
+ *
+ * @example
+ *
+ * randSuperhero({ company: 'Marvel' })
+ *
+ * @example
+ *
+ * randSuperheroName({ length: 10 })
+ *
+ */
+export function randSuperheroName<Options extends SuperheroOptions>(
+  options?: Options
+) {
+  const factory: () => string = () => {
+    const heroes = data
```

Comment:
Why do you filter if the user doesn't pass a company? Use the whole data directly.